README.md

```diff
@@ -26,6 +26,22 @@ Auth0 helps you to:
 1. Go to [Auth0](https://auth0.com/signup) and click Sign Up.
 2. Use Google, GitHub or Microsoft Account to login.
 
+### Auth0 Configuration for the Sample Application(s)
+The example application(s) require to be configured with your Auth0 information in order to run succesfully.
+This can be done by renaming `auth_config.json.example` (https://github.com/auth0-samples/auth0-angular-samples/blob/master/Sample-01/auth_config.json.example) to `auth_config.json` and replacing `{DOMAIN}`, `{CLIENT_ID}` and `{API_IDENTIFIER}` with your tenant specific values:
+
+```
+{
+  "domain": "YOUR_DOMAIN.auth0.com",
```

Comment:
I think if you want to use the full domain here as a placeholder, `YOUR_DOMAIN` would technically be `YOUR_TENANT`. Either that or just use `"domain": "YOUR_DOMAIN"`.

Reply:
Updated